x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ccr/action/DeleteAutoFollowPatternAction.java

```diff
@@ -35,6 +35,9 @@ public AcknowledgedResponse newResponse() {
 
         private String name;
 
+        public Request() {
```

Comment:
I think we should introduce a ctor with parameter `name` instead of a no-args ctor, then make the field `name` final.

Reply:
done: ce3b9fdda8eeceaead058fa37c28fe35a3c125eb